ShowerShow/Service/UserService.cs

```diff
@@ -0,0 +1,26 @@
+﻿using AutoMapper;
+using ShowerShow.DAL;
+using ShowerShow.DTO;
+using ShowerShow.Models;
+using ShowerShow.Utils;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace ShowerShow.Service
+{
+    internal class UserService
```

Comment:
Are you gonna use dependency injection? because you have a service that communicate with the db. what about repository layer?

Reply:
I will change that implementation.